cloudkeeper-one.gemspec

```diff
@@ -36,6 +36,7 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency 'settingslogic', '~> 2.0'
   spec.add_runtime_dependency 'activesupport', '~> 4.0'
   spec.add_runtime_dependency 'tilt', '~> 2.0'
+  spec.add_runtime_dependency 'opennebula', '~> 5.2'
```

Comment:
Well, it used to print something along the lines of "warning: dependency may be too strict". Which is not necessarily a problem, just annoying.